app/src/main/java/com/glia/exampleapp/Utils.java

```diff
@@ -126,6 +130,14 @@ private static ButtonConfiguration getNegativeButtonTestingConfiguration(Context
                 .build();
     }
 
+    private static TextConfiguration getChoiceCardContentTextConfiguration(Context context) {
```

Comment:
looks unused?

Reply:
Example code how to use the textConfiguration, same as with bubbles